### PR TITLE
Remove support for 3.6

### LINKED ISSUE #118: Deprecate 3.6
3.6 support was removed from in 1.13.0, remove it here too.

### PR DESCRIPTION
**Pull request by ocelotl**
Fixes #118

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -36,13 +36,13 @@ jobs:
       run: nox -s test-${{ matrix.python-version }}
 
     - name: Run coverage test
-      if: matrix.python-version == 3.8
+      if: matrix.python-version == 3.7
```
Comment (codeboten) [inline .github/workflows/build.yml:39]: not sure if this was intentional, it used to test the latest version and now this will be testing with the oldest version

File: noxfile.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ def lint(session):
     session.run("pylint", "src")
 
 
-@session(python=["3.6", "3.7", "3.8", "3.9"], reuse_venv=True)
+@session(python=["3.8"], reuse_venv=True)
```
Comment (codeboten) [inline noxfile.py:25]: why the change here?

File: noxfile.py [hunk 3]
```diff
@@ -12,7 +12,7 @@ def test(session):
         session.run("pytest", "tests/test_configuration.py")
 
 
-@session(python=["3.9"], reuse_venv=True)
+@session(python=["3.8"], reuse_venv=True)
```
Comment (codeboten) [inline noxfile.py:15]: why test 3.8 specifically?

File: setup.cfg [hunk 4]
```diff
@@ -34,7 +34,7 @@ classifiers =
     Programming Language :: Python :: 3.10
 
 [options]
-python_requires = >=3.6
+python_requires = >=3.7
```
Comment (codeboten) [inline setup.cfg:37]: need to remove the classifier as well

File: .github/workflows/build.yml (hunk 1 above)
Comment (ocelotl) [reply .github/workflows/build.yml:39]: Yes, I think the version we use for coverage is not relevant and int this way we don't have to update it here so frequently.

File: setup.cfg (hunk 4 above)
Comment (ocelotl) [reply setup.cfg:36]: done

File: noxfile.py (hunk 2 above)
Comment (ocelotl) [reply noxfile.py]: I don't see the point in running`coverage` in many versions, changed it to the oldest one supported.

File: noxfile.py (hunk 3 above)
Comment (ocelotl) [reply noxfile.py]: Changed it to the oldest one so we have to update this less frequently

**Review by codeboten — APPROVED** [review]
